src/main/java/org/broadinstitute/consent/http/db/mapper/RowMapperHelper.java

```diff
@@ -121,8 +118,7 @@ default DataAccessRequestData translate(String darDataString) {
       try {
         data = DataAccessRequestData.fromString(escapedDataString);
       } catch (JsonSyntaxException | NullPointerException e) {
-        String message = "Unable to parse Data Access Request; error: " + e.getMessage();
-        log.error(message);
+        logDebug("Unable to parse Data Access Request; error: %s".formatted(e.getMessage()));
```

Comment:
should this be logException or do we want to change it to debug?

Reply:
This is intentional too - we don't want to spam prod sentry with this kind of error.